packages/connect-voting/subgraph/src/Voting.ts

```diff
@@ -118,6 +119,16 @@ export function updateVoteState(votingAddress: Address, voteId: BigInt): void {
   const vote = VoteEntity.load(buildVoteEntityId(votingAddress, voteId))!
   vote.yea = voteData.value6
   vote.nay = voteData.value7
+  vote.isAccepted = isAccepted(vote.yea, vote.nay, vote.votingPower, vote.supportRequiredPct, vote.minAcceptQuorum, votingApp.PCT_BASE())
 
   vote.save()
 }
+
+function isAccepted(yeas: BigInt, nays: BigInt, votingPower: BigInt, supportRequiredPct: BigInt, minimumAcceptanceQuorumPct: BigInt, pctBase: BigInt): boolean {
+  return hasReachedValuePct(yeas, yeas.plus(nays), supportRequiredPct, pctBase) &&
```

Comment:
```suggestion
  return hasReachedValuePct(yeas, yeas.plus(nays).abs(), supportRequiredPct, pctBase) &&

Reply:
Hey @0xGabi. What's the rationale behind using `abs()`?. We're always adding two positive numbers here.

Reply:
Forgot what I said. I got confused 🙃